### PR TITLE
texinfo: move commands to $prefix/bin and update for secondaryArch perl

### PR DESCRIPTION
**Pull request by korli**
to drop the dependency on libperl_x86.
installs binaries in $prefix/bin on x86.

### CODE REVIEW

**Comment by jmairboeck** [comment]
So you want to keep perl on gcc2 only on x86 and remove perl_x86 again? That would also be solution for #9394, but would also exclude frozen-bubble from there, and maybe other things in the future.

**Comment by korli** [comment]
Actually I have no preferences on perl on x86. AFAICT newer perl versions continue to build on x86_gcc2. perl_x86 only is an option also.

**Comment by Begasus** [comment]
Good to merge?

**Comment by jmairboeck** [comment]
It looks like we are moving perl to gcc 13 only, dropping gcc 2 (see #9439), so the "moving the binaries" part of this is good, the "disable perl xs" part is then not needed any more.

File: sys-apps/texinfo/texinfo-7.0.3.recipe [hunk 1]
```diff
@@ -49,24 +59,31 @@ BUILD_PREREQUIRES="
...
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 
-	# remove libtool files
-	rm $libDir/texinfo/*.la
+	if [ -z "$secondaryArchSuffix" ]; then
```
Comment (jmairboeck) [inline sys-apps/texinfo/texinfo-7.0.3.recipe:84]: why is this if needed?

**Comment by jmairboeck** [comment]
> Good to merge?

Now I think it is ;)

**Comment by jmairboeck** [comment]
I also included a small fix for texinfo-6 (revbump and removed dependency) to rebuild it without perl XS, now that gcc2 perl is gone.

**Comment by jmairboeck** [comment]
I added another small fix for texinfo 7, adding back the $secondaryArchSuffix for libperl (see 14c4afa for the explanation).

**Pull request merged**